packages/oasis-actions/src/operations/aave/close.ts

```diff
@@ -83,7 +77,7 @@ export async function closeStEth(
   })
 
   const returnFunds = actions.common.returnFunds({
-    asset: addresses.ETH,
+    asset: args.debtTokenIsEth ? ADDRESSES.main.ETH : args.debtTokenAddress,
   })
```

Comment:
What should be passed as `debtTokenAddress` when debt token is ETH?  is it '0xeeee...'?

Reply:
it's actually just WETH because the Operations are now written in such a way that we only have to think about WETH given if we send ETH we just wrap it right away. If we're depositing ETH we just deposit WETH. Means we don't have to mess around with AAVEWethGateway either.